pub/index.php

```diff
@@ -10,7 +10,7 @@
 use Magento\Framework\App\Filesystem\DirectoryList;
 
 try {
-    require realpath(__DIR__) . '/../app/bootstrap.php';
```

Comment:
Passing whole string to `realpath` is a bad idea from debugging perspective, see http://stackoverflow.com/a/21035675.

I saw a lot `realpath` call all over the code, but for `__FILE__` or `__DIR__` it seems useless.